src/main/java/uk/gov/mca/beacons/service/model/Beacon.java

```diff
@@ -52,6 +55,7 @@
   private LocalDateTime createdDate;
 
   @Transient
+  @NotEmpty
```

Comment:
Ensures that the list is not empty when validating the JSON payload sent to the rest controller.

There are integration tests to check this case.